internal/oci/container.go

```diff
@@ -448,13 +460,30 @@ func (c *Container) verifyPid() error {
 }
 
 // getPidStartTime reads the kernel's /proc entry for stime for PID.
-func getPidStartTime(pid int) (int, error) {
-	var st unix.Stat_t
-	if err := unix.Stat(fmt.Sprintf("/proc/%d", pid), &st); err != nil {
-		return 0, errors.Wrapf(ErrNotFound, err.Error())
+// inspiration for this function came from https://github.com/containers/psgo/blob/master/internal/proc/stat.go
+// some credit goes to the psgo authors
+func getPidStartTime(pid int) (string, error) {
+	data, err := ioutil.ReadFile(fmt.Sprintf("/proc/%d/stat", pid))
+	if err != nil {
+		return "", errors.Wrapf(ErrNotFound, err.Error())
+	}
+	// The command (2nd field) is wrapped in ().
+	// Filter it out
+	commEnd := strings.LastIndexByte(string(data), ')')
```

Comment:
this is the correct way to do that :+1:

Reply:
but it's better to use `bytes.LastIndexByte` as you don't have to convert to the string.

Reply:
done, and I changed it all to use []byte